Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -811,7 +831,11 @@ private func navigation(fromPushEnvelope envelope: PushEnvelope) -> Navigation?
   }
 
   if let message = envelope.message {
-    return .messages(messageThreadId: message.messageThreadId)
+    if envelope.forCreator == true {
```

Comment:
I used this style because of the original code on lines 793,  872 uses it. I prefer the style that you sagest but wanted to keep consistent with the existing code.